src/module/lib_handle.cc

```diff
@@ -54,8 +54,13 @@ auto LibHandle::Hrtime(MaybeLocal<Array> maybe_diff) -> Local<Value> {
 auto LibHandle::PrivateSymbol(MaybeLocal<String> maybe_name) -> Local<Value> {
 	Local<String> name{};
 	if (maybe_name.ToLocal(&name)) { /* nothing */ }
+	union {
+		Local<Private> *symbol;
+		Local<Value> *value;
+	} cast;
 	auto symbol = Private::New(Isolate::GetCurrent(), name);
-	return *reinterpret_cast<Local<Value>*>(&symbol);
+	cast.symbol = &symbol;
```

Comment:
I think I'd rather leave the warning actually. This change plasters over the warning effectively but the issue still remains. Accessing a union member by a type which is not the most recently previously set type is still UB per specification. This is something I added for xxscreeps because I didn't want to make a new npm module with just this functionality: https://github.com/laverdet/xxscreeps/blob/52e4cb07b5aad57da1a1654ffec17e47f1aa493b/src/driver/private/symbol/isolated-vm.ts

I can't imagine anyone else is using this method.

Reply:
Reverted.